### PR TITLE
Use white vector logo in docs

### PR DESCRIPTION
**Pull request by paulgessinger**
Adds an SVG with a ~~black and~~ white version of the logo. Modifies the build to use that, and include a minimal custom style sheet to make it larger.

### CODE REVIEW

**Comment by msmk0** [comment]
Do we really need the custom CSS? How much does this change the look? I would prefer not to touch the style at all if we can avoid it.

File: docs/figures/acts_logo_b_w.svg [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (msmk0) [inline docs/figures/acts_logo_b_w.svg:1]: Since the logo is white-only the file name should be `acts_logo_white.svg` to avoid confusion.

Comment (paulgessinger) [reply docs/figures/acts_logo_b_w.svg]: Sure, done.

**Comment by paulgessinger** [comment]
> Do we really need the custom CSS? How much does this change the look? I would prefer not to touch the style at all if we can avoid it.

SVG does not have a native size, so the CSS is needed. The custom CSS change is minimal, and should not cause any trouble. I've attached screenshots of the effect the CSS has.

Screen Shot 2020-05-19 at 15 26 08
Screen Shot 2020-05-19 at 15 27 33

**Pull request merged**